packages/airnode-admin/src/utils.ts

```diff
@@ -0,0 +1,15 @@
+export const encaseMnemonic = (mnemonic: string) => {
```

Comment:
Nit: maybe change the file name to `mnemonic` or `mnemonic-utils`, since utils should hold generic functions.